templates/deno/app/root.tsx

```diff
@@ -1,13 +1,13 @@
-import * as React from "react";
+import React from "./deps/react.ts";
 import {
   Links,
   LiveReload,
   Meta,
   Outlet,
   Scripts,
   ScrollRestoration,
-} from "remix";
-import type { MetaFunction } from "remix";
+} from "./deps/@remix-run/react.ts";
+import type { MetaFunction } from "./deps/@remix-run/server-runtime.ts";
```

Comment:
This should import from `@remix-run/deno`. remix-deno should re-export anything it needs from remix-server-runtime. remix-server-runtime is just a common dep of remix-node, remix-cloudflare (future package) and remix-deno.

Reply:
I've got a separate task to make the `@remix-run/*` packages present a complete server runtime interface, so I can address this for `remix-deno` then.